waku/node/waku_node.nim

```diff
@@ -914,7 +914,7 @@ proc mountLightPush*(
 
       if publishedCount == 0:
         ## Agreed change expected to the lightpush protocol to better handle such case. https://github.com/waku-org/pm/issues/93
-        debug "Lightpush request has not been published to any peers"
+        info "Lightpush request has not been published to any peers"
```

Comment:
Shouldn't we log the hash here? It would be hard to trace otherwise with many messages going in parallel.

Reply:
Thanks I will submit a change with that